src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,224 @@
+    <item name="WrongPowerSupply" value="0x0D"/>
+    <item name="LiveNeutralSwap" value="0x0E"/>
+    <item name="OverTemperature" value="0x0F"/>
+    <item name="Other" value="0xFF"/>
+  </enum>
+  <enum name="EnergyTransferStoppedReasonEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="EVStopped" value="0x00"/>
+    <item name="EVSEStopped" value="0x01"/>
+    <item name="Other" value="0x02"/>
+  </enum>
+  <bitmap name="TargetDayOfWeekBitmap" type="bitmap8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <field name="Sunday" mask="0x01"/>
+    <field name="Monday" mask="0x02"/>
+    <field name="Tuesday" mask="0x04"/>
+    <field name="Wednesday" mask="0x08"/>
+    <field name="Thursday" mask="0x10"/>
+    <field name="Friday" mask="0x20"/>
+    <field name="Saturday" mask="0x40"/>
+  </bitmap>
+  <struct name="ChargingTargetStruct" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item fieldId="0" name="TargetTime" type="int16u" min="0" max="1439"/>
+    <item fieldId="1" name="TargetSoC" type="percent" optional="true"/>
+    <item fieldId="2" name="AddedEnergy" type="int64s" min="0" max="500000000" optional="true"/>
+  </struct>
+  <cluster apiMaturity="provisional">
+    <name>Energy EVSE</name>
+    <domain>Energy Management</domain>
+    <code>0x0099</code>
+    <define>ENERGY_EVSE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>Electric Vehicle Supply Equipment (EVSE) is equipment used to charge an Electric Vehicle (EV) or Plug-In Hybrid Electric Vehicle. This cluster provides an interface to the functionality of Electric Vehicle Supply Equipment (EVSE) management.</description>
+    
+    <globalAttribute side="server" code="0xFFFD" value="2" />
+
+    <!--Attributes-->
+    <attribute code="0x0000" side="server" type="StateEnum" define="STATE" isNullable="true" writable="false" optional="false">State</attribute>
+    <attribute code="0x0001" side="server" type="SupplyStateEnum" define="SUPPLY_STATE" writable="false" optional="false">SupplyState</attribute>
+    <attribute code="0x0002" side="server" type="FaultStateEnum" define="FAULT_STATE" writable="false" optional="false">FaultState</attribute>
+    <attribute code="0x0003" side="server" type="epoch_s" define="CHARGING_ENABLED_UNTIL" isNullable="true" default="0" max="0xFFFFFFFE" writable="false" optional="false">ChargingEnabledUntil</attribute>
+    <!--Conformance feature V2X - for now optional-->
+    <attribute code="0x0004" side="server" type="epoch_s" define="DISCHARGING_ENABLED_UNTIL" isNullable="true" default="0" max="0xFFFFFFFE" writable="false" optional="true">DischargingEnabledUntil</attribute>
+    <attribute code="0x0005" side="server" type="int64s" define="CIRCUIT_CAPACITY" default="0" min="0" max="80000" writable="false" optional="false">CircuitCapacity</attribute>
```

Comment:
Spec has the "amperage-mA" type here, but I don't see a definition of it in the spec anywhere.... so I can't tell whether this type is right.

Reply:
Created new issue: https://github.com/project-chip/connectedhomeip/issues/30665